connaisseur/__main__.py

```diff
@@ -29,5 +29,5 @@
     # container
     APP.run(
         host="0.0.0.0",  # nosec
-        ssl_context=("/etc/certs/tls.crt", "/etc/certs/tls.key"),
+        ssl_context=("/app/certs/tls.crt", "/app/certs/tls.key"),
```

Comment:
I doubt this change is necessary for adding modular validation. Still, it is in this commit. I really *urge* everyone to keep changes minimal, and at least keep commit size minimal. If you want to change some directory structure, that should go into a separate commit.

Commits should not exceed let's say 200 lines.

Reply:
It's really hard to review commits like these, when you have to keep lots of different aspects in mind. It's unnecessarily hard.

Reply:
Ideally, this PR would start with a refactor commit that does *nothing* but modularizing validation under the hood. For this commit, you would, ideally, not have to touch any test cases.

Then, another commit that introduces the necessary syntax changes in the configuration. This of course would need to touch tests.

Then, another one that adds new validators.